datasets/snli/README.md

```diff
@@ -0,0 +1,140 @@
+---
+language: 
+- en
+task:
+- text-classification
+purpose:
+- NLI
+size:
+- ">100k"
+language producers:
+- crowdsourced
+annotation:
+- crowdsourced
+tags:
+- extended-from-other-datasets
+license: "CC BY-SA 4.0"
+---
+# Dataset Card for SNLI
+
+## Table of Contents
+- [Tasks Supported](#tasks-supported)
+- [Purpose](#purpose)
+- [Languages](#languages)
+- [People Involved](#who-iswas-involved-in-the-dataset-use-and-creation)
+- [Data Characteristics](#data-characteristics)
+- [Dataset Structure](#dataset-structure)
+- [Known Limitations](#known-limitations)
+- [Licensing information](#licensing-information)
+
+## Tasks supported:
+### Task categorization / tags
+
+Text to structured, three-way text classification
```

Comment:
What makes this structured? That would usually make me think of structured prediction—i.e., predicting something more complex than a label from a small fixed set.

Reply:
It was in contrast with text-to-text, but we're still working on the tag set. Is there something you think would fit better?

Reply:
Just cut 'structured'?